eng/pipelines/prepare-release.yml

```diff
@@ -13,16 +13,16 @@ stages:
           displayName: 'Publish Release Drop'
           condition: succeeded()
     variables:
-    - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release/')) }}:
+    - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), or(startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), startsWith(variables['Build.SourceBranch'], 'refs/heads/reltest/'))) }}:
       - group: DotNet-Diagnostics-Storage
-      - group: DotNet-DotNetStage-Storage
-      - group: Release-Pipeline
+      - group: DotNetBuilds storage account read tokens
```

Comment:
`group` can have spaces?  Is the name of the group `DotNetBuilds` here ?

Reply:
yeah, group names can have spaces. It's the whole string